pkg/k8s/client/cell.go

```diff
@@ -545,6 +587,86 @@ func FakeClientBuilder() ClientBuilderFunc {
 	}
 }
 
+func FakeClientCommand(fc *FakeClientset) script.Cmd {
+	return script.Command(
+		script.CmdUsage{
+			Summary: "interact with fake k8s client",
+			Args:    "<command> args...",
+		},
+		func(s *script.State, args ...string) (script.WaitFunc, error) {
+			if len(args) < 1 {
+				return nil, fmt.Errorf("usage: k8s <command> files...\n<command> is one of add, update or delete.")
+			}
+
+			action := args[0]
+			if len(args) < 2 {
+				return nil, fmt.Errorf("usage: k8s %s files...", action)
+			}
+
+			for _, file := range args[1:] {
+				b, err := os.ReadFile(s.Path(file))
+				if err != nil {
+					// Try relative to current directory, e.g. to allow reading "testdata/foo.yaml"
+					b, err = os.ReadFile(file)
+				}
+				if err != nil {
+					return nil, fmt.Errorf("failed to read %s: %w", file, err)
+				}
+				obj, gvk, err := testutils.DecodeObjectGVK([]byte(b))
```

Comment:
`[]byte` is redundant here 
```suggestion
				obj, gvk, err := testutils.DecodeObjectGVK(b)
```